Controller/Redirect/Process.php

```diff
@@ -348,7 +354,25 @@ public function execute()
         $this->logger->addDebug(__METHOD__ . '|9|');
         return $this->_response;
     }
-
+    /**
+     * Set flag if user is on the payment provider page
+     *
+     * @param OrderPaymentInterface $payment
+     *
+     * @return void
+     */
+    protected function setPaymentOutOfTransit(OrderPaymentInterface $payment)
+    {
+        $payment->setAdditionalInformation(AbstractMethod::BUCKAROO_PAYMENT_IN_TRANSIT, false);
+
+        //set in the payment session as well
+        $this->_objectManager->get(
```

Comment:
object manager should not be used, instantiate proper session class via constructor